disnake/user.py

```diff
@@ -297,7 +297,7 @@ class ClientUser(BaseUser):
 
         .. describe:: str(x)
 
-            Returns the user's username (with discriminator, if not migrated to new system yet).
+            Returns the user's username (with discriminator).
```

Comment:
this is now wrong, as it won't include the discrim if the account has been migrated

Reply:
from the looks of it, bot accounts (i.e. `ClientUser`s) won't be migrated any time soon, but this seems fine
https://github.com/DisnakeDev/disnake/pull/1044/commits/95d0f2470fb3a990b0bacb7966fcf1120d3746dd